core/components/molecules/tooltip/__stories__/ShowTooltip.story.jsx

```diff
@@ -0,0 +1,41 @@
+import { Tooltip } from '@/index';
+
+export const showTooltip = () => {};
+
+const customCode = `() => {
+  const [showTooltip, setShowTooltip] = React.useState(false);
+
+  return (
+    <div className="Row align-items-center">
+      <div>
+        <Tooltip showTooltip={showTooltip} tooltip="An awesome tooltip">
+          <Button>Conditional Tooltip</Button>
+        </Tooltip>
+      </div>
+      <div className="ml-6">
+        <Label className="mr-4">Show Tooltip</Label>
+        <Switch
+          value={showTooltip}
+          aria-label="Toggle tooltip"
+          size="tiny"
+          onChange={(_, selected) => setShowTooltip(selected)}
+        />
+      </div>
+    </div>
```

Comment:
Can we please avoid using multiple div and simply try to align it vertically to make the code more readable.

```
<div>
        <div className="mb-6 d-flex">
          <Label className="mr-4">Show Tooltip</Label>
          <Switch
            value={showTooltip}
            aria-label="Toggle tooltip"
            size="tiny"
            onChange={(_, selected) => setShowTooltip(selected)}
          />
        </div>
  
        <Tooltip showTooltip={showTooltip} tooltip="An awesome tooltip">
          <Button>Conditional Tooltip</Button>
        </Tooltip>
      </div>
```